checks/fuzzing_test.go

```diff
@@ -69,11 +75,10 @@ func TestFuzzing(t *testing.T) {
 				},
 			},
 			wantErr: false,
-			want:    checker.CheckResult{Score: 10},
 			expected: scut.TestReturn{
-				NumberOfWarn:  0,
+				NumberOfWarn:  6,
```

Comment:
This seems to be a common pattern these probes will run into. When you have `n` tool/fuzzer/foo options, getting `n-1` warnings for the "negative" outcomes of all the others.

Is it worth thinking about how to handle this?

Reply:
what do you mean by "handle this"?